### PR TITLE
Editing Example 0

### PR DESCRIPTION
**Pull request by ttewah**

### CODE REVIEW

File: students/5/modern-js.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
+// File: modern-js.js
+// Date: 5/30/2022
+// Desc: Modernize this legacy JavaScript:
+//       - Replace ALL functions with arrow functions (this includes the sort callback function)
+//       - Replace var with let or const.
+//
+
+const displayResults = (personList, count) => {
+
```
Comment (mburolla) [inline students/5/modern-js.js:10]: Let's remove this whitespace.

File: students/5/modern-js.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//
+// File: modern-js.js
+// Date: 5/30/2022
+// Desc: Modernize this legacy JavaScript:
+//       - Replace ALL functions with arrow functions (this includes the sort callback function)
+//       - Replace var with let or const.
+//
+
+const displayResults = (personList, count) => {
```
Comment (mburolla) [inline students/5/modern-js.js:9]: Does count really need to be in this function signature?

File: students/5/modern-js.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      { firstName: "Charlie", lastName: "Thumb", age: 5 },
+    ];
+  }
+  return retval;
+}
+
+const main = () => {
+  let table = "person";
+  let personList = getData(table);
+  personList.sort(function(a,b){return a.age - b.age});
```
Comment (mburolla) [inline students/5/modern-js.js:31]: Don't forget to convert this to an arrow function.

File: students/5/modern-js.js (hunk 1 above)
Comment (ttewah) [reply students/5/modern-js.js]: issue resolved

File: students/5/modern-js.js (hunk 2 above)
Comment (ttewah) [reply students/5/modern-js.js]: issue resolved

**Comment by ttewah** [comment]
Sir I am waiting on your review and a confirmation to merge pull request. Thanks.

**Comment by mburolla** [comment]
👍

**Pull request merged**